indico/web/client/styles/design_system/README.txt

```diff
@@ -0,0 +1,157 @@
+Indico Design System
```

Comment:
I think this file would deserve being markdown as well :)

Reply:
And just thinking, would it make sense to move the design system in a separate PR? I think both things are pretty much ready to merge, but it might be nicer to have them in separate PRs (also because IMHO the design system deserves a changelog entry in 'internal changes')

Reply:
> And just thinking, would it make sense to move the design system in a separate PR?

To me this type of decision is usually based on the need to revert them separately. I've no strong opinion either way.

Reply:
Same here (no strong opinion). If it's already a separate commit I'd split it, if not I wouldn't bother to spend/waste time with it ;)